lighthouse-core/audits/metrics/first-contentful-paint-3g.js

```diff
@@ -29,12 +29,11 @@ class FirstContentfulPaint3G extends Audit {
    */
   static get defaultOptions() {
     return {
-      // 25th and 5th percentiles HTTPArchive on Fast 3G -> multiply by 1.5 for RTT differential -> median and PODR
-      // p10 is then derived from them.
+      // 25th and 8th percentiles HTTPArchive on Slow 4G -> multiply by 1.5 for RTT differential -> median and p10.
       // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2018_04_01_mobile?pli=1
-      // https://www.desmos.com/calculator/fflcrsn9sj
-      p10: 3504,
-      median: 6000,
+      // https://www.desmos.com/calculator/xi5oympawp
+      p10: 2700,
+      median: 4500,
```

Comment:
these are updated based on the 1.5x multiplier mentioned above applied to the updated control points